packaging/hudi-flink-bundle/pom.xml

```diff
@@ -221,6 +221,33 @@
                   <pattern>com.fasterxml.jackson.</pattern>
                   <shadedPattern>${flink.bundle.shade.prefix}com.fasterxml.jackson.</shadedPattern>
                 </relocation>
+                <!-- The classes below in org.apache.flink.table.data package come from flink-table-runtime,
+                which have to be shaded one by one,
+                instead of shading all classes under org.apache.flink.table.data. -->
+                <relocation>
+                  <pattern>org.apache.flink.table.data.ColumnarRowData</pattern>
+                  <shadedPattern>${flink.bundle.shade.prefix}org.apache.flink.table.data.ColumnarRowData</shadedPattern>
```

Comment:
Oops, the fix is ugly, we may need to find better solution ~

Reply:
Okey, I will find a better way